schemas/v1.0/azure.yaml.json

```diff
@@ -135,9 +140,125 @@
                                 "description": ""
                             }
                         }
+                    },
+                    "scripts": {
+                        "type": "object",
+                        "title": "Definition of command hooks for developer integration",
+                        "descriptions": "Command hooks should match `azd` command names prefixed with `pre` or `post` depending on when the script should execute relative to the command",
+                        "minProperties": 1,
+                        "additionalProperties": {
+                            "type": "object",
+                            "additionalProperties": false,
+                            "properties": {
+                                "type": {
```

Comment:
Could we get rid of this property and infer the type of the script from the extension? E.g. 
- `.sh` or `.bash` => bash
- `.ps1` => PowerShell
- something else => we do not know, but we can try executing it directly via default shell

Reply:
It is deferred based on the file extension.  Going to keep the property overall to support inline mode.